### ISSUE TITLE
Breaking changes after upgrading to 5.1.0

### ISSUE THREAD

**Issue by born4new** [sunaku/md2man#22]
I made a folder hierarchy as follows:

```
.
└── man
    └── test_markdown.md
```

When I launch the following command in 5.0.3 from the root

`md2man-rake md2man:man`

the tools works great:

```
.
└── man
    ├── test_markdown
    └── test_markdown.md
```

However, the same command from the same place with 5.1.0 breaks:

```ruby
/opt/rubies/2.2.0/lib/ruby/2.2.0/rake/invocation_chain.rb:16:in `append': Circular dependency detected: TOP => md2man:man => man/test_markdown => man => md2man:man (RuntimeError)
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/invocation_chain.rb:28:in `append'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/task.rb:171:in `invoke_with_call_chain'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/task.rb:201:in `block in invoke_prerequisites'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/task.rb:199:in `each'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/task.rb:199:in `invoke_prerequisites'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/task.rb:178:in `block in invoke_with_call_chain'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/monitor.rb:211:in `mon_synchronize'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/task.rb:172:in `invoke_with_call_chain'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/task.rb:201:in `block in invoke_prerequisites'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/task.rb:199:in `each'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/task.rb:199:in `invoke_prerequisites'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/task.rb:178:in `block in invoke_with_call_chain'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/monitor.rb:211:in `mon_synchronize'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/task.rb:172:in `invoke_with_call_chain'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/task.rb:201:in `block in invoke_prerequisites'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/task.rb:199:in `each'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/task.rb:199:in `invoke_prerequisites'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/task.rb:178:in `block in invoke_with_call_chain'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/monitor.rb:211:in `mon_synchronize'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/task.rb:172:in `invoke_with_call_chain'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/task.rb:165:in `invoke'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/application.rb:150:in `invoke_task'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/application.rb:106:in `block (2 levels) in top_level'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/application.rb:106:in `each'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/application.rb:106:in `block in top_level'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/application.rb:115:in `run_with_threads'
	from /opt/rubies/2.2.0/lib/ruby/2.2.0/rake/application.rb:100:in `top_level'
	from /opt/rubies/2.2.0/lib/ruby/gems/2.2.0/gems/md2man-5.1.0/bin/md2man-rake:61:in `<main>'

```

**Comment by sunaku** [comment]
:astonished: Could you try running `md2man-rake man` instead?  And what version of Rake are you using?

**Comment by sunaku** [comment]
Oh I see, `man` is both a folder name _and_ a task name. :cold_sweat: I didn't notice this! :disappointed:  Fixed now in a3883b4f14c38e00844e31e9b7c30153a30a50d9.

**Comment by sunaku** [comment]
@born4new Please upgrade to https://github.com/sunaku/md2man/releases/tag/v5.1.1 :gift:
